static/scss/collection.scss

```diff
@@ -158,6 +171,14 @@ $collection-tools-line-height: 36px;
           line-height: $collection-tools-line-height;
           vertical-align: middle;
         }
+
+        i.material-icons {
+          color: $tool-icon-color;
+
+          &:hover {
+            color: $tool-icon-color-hover;
+          }
+        }
```

Comment:
Can you combine these definitions with the definitions for `i` above? It's only there to apply styles to the icon. Result would be something like:

```scss
        i.material-icons {
          line-height: $collection-tools-line-height;
          vertical-align: middle;
          color: $tool-icon-color;

          &:hover {
            color: $tool-icon-color-hover;
          }
        }
```